core/include/traccc/seeding/doublet_finding.hpp

```diff
@@ -18,9 +18,9 @@ namespace traccc {
 
 /// Doublet finding to search the combinations of two compatible spacepoints
 struct doublet_finding
-    : public algorithm<
-          std::pair<host_doublet_collection, host_lin_circle_collection>(
-              const sp_grid&, const sp_location&, const bool&)> {
+    : public algorithm<std::pair<doublet_collection_types::host,
+                                 lin_circle_collection_types::host>(
+          const sp_grid&, const sp_location&, const bool&)> {
```

Comment:
Not for this PR, but wow... We really provide a boolean to the algorithm as a constant reference?!? :scream: We should really fix that in a separate PR.